README.md

```diff
@@ -242,15 +246,33 @@ IRB's default completion `IRB::RegexpCompletor` uses Regexp. IRB has another exp
 
 ### How to Enable IRB::TypeCompletor
 
-To enable IRB::TypeCompletor, run IRB with `--type-completor` option
+Install [ruby/repl_type_completor](https://github.com/ruby/repl_type_completor/) with:
+```
+$ gem install repl_type_completor
+```
+Or add these lines to your project's Gemfile.
+```ruby
+gem 'irb'
+gem 'repl_type_completor', group: :development
```

Comment:
Maybe `:test` group too? Otherwise users won't get the same completion behaviour when adding `binding.irb` in tests?